_topics/en/adaptor-signatures.md

```diff
@@ -0,0 +1,242 @@
+#aliases:
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Privacy Enhancements
+  - Scripts and Addresses
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Adaptor signatures** are auxiliary signature data that commit to a
+  hidden value.  When an adaptor signature is combined with a final
+  signature, it reveals the hidden value.  Alternatively, when combined
+  with the hidden value, it reveals the final signature.  The commitment
+  may be reused in other signatures without the secondary signers
+  knowing the hidden value or their own final signature.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Adaptor signatures never need to be published onchain.   To anyone
+  without a corresponding adaptor signature, a final signature looks
+  like any other digital signature, giving adaptor signatures efficiency
+  and privacy advantages over other onchain locking mechanisms such as
+  hashlocks and preimages.
+
+  ### Example
+
+  The multiple uses of adaptor signatures can be seen in a simple
+  coinswap protocol.  For example, Alice can give Bob an adaptor
+  signature for a transaction that promises to pay him 1 BTC.  An
+  adaptor signature by itself can't be used as a final signature, so
+  Alice hasn't paid Bob yet.
+
+  What the adaptor signature does provide Bob is a commitment to Alice's
+  hidden value.  This commitment is made in the form of a value Bob
+  can use to create a second adaptor signature that commits to the same
+  hidden value as Alice's adaptor signature.  Bob can make that
+  commitment even without knowing Alice's hidden value or his own final
+  signature for that commitment.  Bob gives Alice his adaptor signature
+  and a corresponding transaction that promises to pay her 1 BTC.
+
+  Alice has always known the hidden value, so she can combine the hidden
+  value with Bob's adaptor signature to get his final signature for the
+  transaction that pays her.  She broadcasts the transaction and
+  receives Bob's payment.  When Bob sees that transaction onchain, he
+  can combine its final signature with the adaptor signature he gave
+  Alice, allowing him to derive the hidden value.  Then he can
+  combine that hidden value with the adaptor signature Alice gave him to
+  get her final signature for the first transaction.  Bob broadcasts
+  that transaction to receive Alice's payment, completing the coinswap.
+
+  Besides coinswaps, there are several other [proposed uses][scriptless
+  scripts repo] for adaptor signatures.
+
+  <div class="qa_details">
+  <details markdown="1"><summary>Click to show the same coinswap example in mathematical terms</summary>
+  *In the following example, we assume the use of BIP340-compatible
+  schnorr signatures.  We use lowercase variables for scalars and
```

Comment:
We decided to lowercase schnorr (and other names like merkle) in our style guide: https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#common-nouns

Reply:
Well, I think that's strange but okay...

Reply:
Do you capitalize "algorithm" just because it's named after ["Algorithmi"](https://en.wikipedia.org/wiki/Muhammad_ibn_Musa_al-Khwarizmi)?  :-)  The general rule in most style guides is to stop capitalizing eponyms after they enter common use---and I think we're well beyond the point for a technical publication about Bitcoin where schnorr signatures and various merkle things are in common use.

Reply:
Yeah, idk, I believe it just seems strange to me because noone in academia does this.

If you ask me for an argument, Schnorr and Merkle are living persons. I believe it's somewhat strange to not spell their names as names. On the other hand it's a honor then that their names are now proper words, so there are good arguments for either side. :)